common/src/main/java/org/apache/comet/parquet/ReadOptions.java

```diff
@@ -173,14 +147,24 @@ public Builder(Configuration conf) {
       this.conf = conf;
       this.parallelIOEnabled =
           conf.getBoolean(
```

Comment:
This is reading from the Hadoop conf. If I set the new configs on my Spark context, how would they get propagated to the Hadoop conf?

Reply:
Spark sql copies over configs that are not from spark into the hadoop config when the sql context is created. There are other settings that also use this  ( e.g. `COMET_USE_LAZY_MATERIALIZATION`, `COMET_SCAN_PREFETCH_ENABLED`)